main/solution/backend/config/infra/cloudformation.yml

```diff
@@ -767,6 +738,24 @@ Resources:
               - appstream:UpdateImagePermissions
             Resource:
               - !Sub 'arn:aws:appstream:${AWS::Region}:${AWS::AccountId}:image/ServiceWorkbench*'
+          - Effect: Allow
+            Action:
+              - iam:CreateRole
+            Resource:
+              - !Sub 'arn:${AWS::Partition}:iam::${AWS::AccountId}:role/study-*'
+            Condition:
+              StringLike:
+                iam:PermissionsBoundary: !Sub 'arn:${AWS::Partition}:iam::${AWS::AccountId}:policy/study-*'
```

Comment:
Can we create a permission boundary here which has read/write access to egress bucket and reference it here. It's better to avoid wildcards in permission boundary since there can be an overly permissive policy created which starts with `study-`.

Reply:
Added the permissionBoundary